docs/slo-policy.md

```
@@ -0,0 +1,71 @@
+- Beta or preview features (they're still experimental)
+- Scheduled maintenance 
+- Issues outside our control (internet outages, AWS problems, etc.)
+- Problems you caused (wrong configuration, hitting rate limits, etc.)
+- Third-party service failures
+
+## Need Help?
+
+Here's how to reach us:
+
+- **Support Portal**: [support.gruntwork.io](https://support.gruntwork.io) - Submit tickets and track issues
+- **Email**: support@gruntwork.io - Direct email support
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)